app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+
+
+def calculate_team_total_rating(list_of_players: list[ElfRanger,
+                                                      Druid,
+                                                      DwarfWarrior,
+                                                      DwarfBlacksmith]) -> int:
```

Comment:
you can replace all list types with only one. What do you think what the type of list? Also, change your type

Reply:
I guess I got it. Fixed.